gdal/ogr2ogr.xml

```diff
@@ -0,0 +1,82 @@
+<tool id="ogr2ogr" name="OGR2ogr" version="@VERSIONOGR@">
+    <description>converts simple features data between file formats</description>
+    <macros>
+        <import>gdal_macros.xml</import>
+    </macros> 
+    <requirements>
+        <requirement type="package" version="2.4.0">gdal</requirement>
+        <requirement type="package" version="3.7.1">geos</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        ogr2ogr -f $f '$output' '$input'
+
+    ]]></command>
+    
+    <inputs>
+        <param type="data" name="input" format="data" label="Gdal supported input file" help="http://www.gdal.org/ogr_formats.html"/>
+        <param type="select" name="f" label="Conversion format" value="" help="Select the format to convert your input file. Currently supported : Shapefile, GeoJSON, GML, KML and WFS.">
+            <option value="ESRI Shapefile ">ESRI Shapefile </option>
+            <option value="GeoJSON">GEOJSON</option>
+            <option value="GML">GML</option>
+            <option value="KML">KML</option>
+            <option value="WFS">WFS</option>
+        </param>
+    </inputs>
+    
+    <outputs>
+        <data name="output" auto_format="true" label="${on_string}.${f}"/> 
```

Comment:
For workflows its better to not use auto_format. Because Galaxy knows the format beforehand.
It's better to use https://github.com/galaxyproject/tools-iuc/blob/57493fdc88e6128bed127993464f2a6480e2774d/tools/mothur/make.fastq.xml#L49 instead if you know the format before.

Reply:
Yes, now that we fixed a limited format possibilities this become very handy !

Reply:
Too easy ;)